### DISCUSSION TITLE
Deceptive Pricing

### DISCUSSION THREAD

**Discussion by lusky3** [azukaar/Cosmos-Server#340] [General]
Assuming this is just an oversight, but on [the website](https://cosmos-cloud.io/pricing), the price of Home Premium is being advertised **per year**.
![image](https://github.com/user-attachments/assets/6f0fa854-dee9-42ef-98be-d926a45c48f7)

This means a 1 year subscription costs $7.01, which is not the case. It is $7.01 **per month** with a year commitment.

**Comment by azukaar** [comment]
Thanks sorry about the issue - everytime I ask an AI to change this page it reverts the label to say "per year" when yearly pricing is selected because it does not understand that the page shows the monthly page either way. Changed it back
